packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js

```diff
@@ -32,6 +32,7 @@ export default class DatabaseService {
   static getIcebergHeaders() {
     return {
       headers: {
+        Pragma: 'no-cache',
```

Comment:
Every get requests will then apply those headers.

We are losing the benefit of the cache made by Iceberg so.

Are you sure it's the only way to do this?

Reply:
Yeah I am sure... We dont need the cache in any API call that we make for databases. Because, we might have data changes more frequently in these calls..